metacat-main/src/main/java/com/netflix/metacat/main/services/MetacatServiceHelper.java

```diff
@@ -105,6 +113,10 @@ public void postPreUpdateEvent(
             this.eventBus.postSync(
                 new MetacatSaveTablePartitionPreEvent(name, metacatRequestContext, this, partitionsSaveRequestDto)
             );
+        } else if (name.isViewDefinition()) {
+            this.eventBus.postSync(
+                new MetacatUpdateMViewPreEvent(name, metacatRequestContext, this, (TableDto) dto)
+            );
```

Comment:
I am not sure what this change is for. Is this another functionality besides the timer and delete metadata ?